docs/basics/part-5-importing.md

```diff
@@ -0,0 +1,203 @@
+```
+
+Examples in [Python](#more-examples), [Javascript](#more-examples) and [Java](#more-examples) are at the bottom of this page.
+
+## Calling on Targets From Other Earthfiles
+So far we've seen how the `FROM` command in Earthly has the ability to reference another target's image as its base image, like in the case below where the`+build` target uses the image from the `+deps` target.
+
+```Dockerfile
+VERSION 0.6
+FROM golang:1.15-alpine3.13
+WORKDIR /go-example
+
+deps:
+    COPY go.mod go.sum ./
+    RUN go mod download
+    # Output these back in case go mod download changes them.
+    SAVE ARTIFACT go.mod AS LOCAL go.mod
+    SAVE ARTIFACT go.sum AS LOCAL go.sum
+
+build:
+    FROM +deps
+    COPY main.go .
+    RUN go build -o build/go-example main.go
+    SAVE ARTIFACT build/go-example /go-example AS LOCAL build/go-example
+
+```
+But `FROM` also has the ability to import targets from Earthfiles in different directories. Let's say we have a directory structure like this.
+```bash
+.
+├── services
+|   ├── service-one
+|   |   ├── Earthfile (containing +deps)
+|   |   ├── go.mod
+|   |   ├── go.sum
+├── main.go
+└── Earthfile
+
+```
+To copy the files for [this example ( Part 3 )](https://github.com/earthly/earthly/tree/main/examples/tutorial/go/part5) run
+
+```bash
+earthly --artifact github.com/earthly/earthly/examples/tutorial/go:main+part5/part5 ./part5
+```
+
+We can use a target in the Earthfile in `/service/services` from inside the Earthfile in the rood of our directory.
+
+```Dockerfile
+# `./service/services/Earthfile`
+
+VERSION 0.6
+FROM golang:1.15-alpine3.13
+WORKDIR /go-example
+
+deps:
+    COPY go.mod go.sum ./
+    RUN go mod download
+    # Output these back in case go mod download changes them.
+    SAVE ARTIFACT go.mod AS LOCAL go.mod
+    SAVE ARTIFACT go.sum AS LOCAL go.sum
+```
+
+```Dockerfile
+# ./Earthfile
+build:
+    FROM ./services/service-one+deps
+    COPY main.go .
+    RUN go build -o build/go-example main.go
+    SAVE ARTIFACT build/go-example /go-example AS LOCAL build/go-example
+```
+This code tells `FROM` that there is another Earthfile in  the `services/service-one` directory and that the Earthfile  contains a target called `+deps`. In this case, if we were to run `+build` Earthly is smart enough to go into the subdirectory, run the  `+deps` target in that Earthfile, and then use it as the base image for `+build`.
+
+We can also reference an Earthfile in another repo, which works in a similar way.
+
+```Dockerfile
+build:
+    FROM github.com/example/project+remote-target
+    COPY main.go .
+    RUN go build -o build/go-example main.go
+    SAVE ARTIFACT build/go-example /go-example AS LOCAL build/go-example
+```
+
+## Importing Whole Projects
+In addition to importing tagets from other files, we can also import an entire file with the `IMPORT` command.
+
+```Dockerfile
+VERSION 0.6
+FROM golang:1.15-alpine3.13
+WORKDIR /go-example
+IMPORT ./service AS my_service
```

Comment:
(Just a matter of taste, since the IMPORT becomes global anyway in this case)

```suggestion
VERSION 0.6
IMPORT ./service AS my_service
FROM golang:1.15-alpine3.13
WORKDIR /go-example
```